### DISCUSSION TITLE
유니티에서 본인이 만든 맵 실행하고 본인 캐릭터도 실행하는 방법 아시나요?

### DISCUSSION THREAD

**Discussion by phy9051** [naverz/zepeto-studio-kor#597] [5. Q&A]
유니티에, 제페토 스튜디오에서 제공하는 샘플맵 열고 캐릭터 뜨는 건 되는데

만든 맵을 불러오면 캐릭터가 안 떠요.

혹시 방법 아시는 분 있나요? ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 새로운 맵을 만들때는 아래 가이드에 따라 캐릭터를 불러올 수 있습니다.
https://studio.zepeto.me/kr/guides/character-control

**Reply by phy9051** [reply]
감사합니다!!
